modules/ROOT/pages/ogm/installation.adoc

```diff
@@ -1,6 +1,7 @@
 [[ogm-installation]]
 :description: This page describes how to install the OGM in Neo4j GraphQL and includes examples of how to use it.
 = Installation
+:page-aliases: ogm/examples/index.adoc, omg/examples/rest-api.adoc, omg/examples/custom-resolvers.adoc
```

Comment:
https://neo4j.com/docs/graphql/3/ogm/examples/custom-resolvers/ is already listed in the three versions and it's taking to https://neo4j.com/docs/graphql/5/custom-resolvers/ . We need to remove it from here or it will cause a loop.

omg/examples/rest-api.adoc doesn't have a substitute, so I wouldn't redirect to /ogm/installation.adoc

All in all, I think we could go only with ogm/examples/index.adoc

Reply:
true and true, updated